.github/workflows/presubmit.yml

```
@@ -0,0 +1,88 @@
+name: presubmit
+on:
+  push:
```

```suggestion
  push:
    branches: ['main']
```

Let's keep a "lazy" CI on secondary branches; those would need to opt into CI by having a PR around them 🙂